exercises/5_OOP-and-Intents/oop/src/ClarinetPlayer.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Created by c4q-rosmary on 4/30/15.
+ */
+
+//Create the concrete class ClarinetPlayer, which extends Musician.
+public class ClarinetPlayer<T> extends Musician
+{
+    protected Clarinet clarinet;
+
+
+    //The ClarinetPlayer constructor should take in a Clarinet, and save it in a field called myClarinet
+    public ClarinetPlayer (Clarinet clarinet) {
+        this.clarinet = clarinet;
```

Comment:
Use the super constructor and add a clarinet. You still need to build a Musician object